### ISSUE TITLE
Inference on sample image

### ISSUE THREAD

**Issue by remy-byte** [kyotovision-public/fumet#4]
Hello! I'm coming back with a question.
Here I have a snippet of an inference code where I tried using a pretrained model to run an inference on a sample image following the evaluation scripts. 
I know that from the reading of the image to actually passing it trough the model, there are also some functions in the dataset for aligning the image size with the focal lengths and so on, but I wanted to firstly get a "reasonable result" as I know the scales of the depth are not going to be correctly scaled.
I tried it w/o the post_process settings, that flips the image and concatenates it and i get some wierd results in the pred_depth.
The output I am getting is a black map with some white dots. I checked the values, and those range from 0.1  to 80 as those were the MIN/MAX values I've set. The thing is, most of the pixels are close to 0.
Is there a missing step piece I didn't do? I also checked the reading of the images in BaseEval, and there weren't any normalization steps there either.

```import os
import random
from pathlib import Path

import cv2
import numpy as np
import torch
from torch.utils.data import DataLoader
from tqdm import tqdm

import datasets
import networks
from eval_utils import batch_post_process_disparity, compute_errors
from layers import disp_to_depth
from options import Options

MIN_DEPTH = 1e-3
MAX_DEPTH = 80

weights_dir = (
    "....................../weights_5/"
)
print("-> Loading model from ", weights_dir)
encoder_path = weights_dir + "encoder.pth"
depth_decoder_path = weights_dir + "depth.pth"
print(encoder_path, depth_decoder_path)
encoder = networks.ResnetEncoder(50, False)
depth_decoder = networks.DepthDecoder(encoder.num_ch_enc, [0, 1, 2, 3])

img = "............./FUMET/fumet/berlin_000001_000019_leftImg8bit.png"
print("Loading pretrained encoder")
img = cv2.imread(img, -1)
img = cv2.cvtColor(img, cv2.COLOR_BGR2RGB)
img = cv2.resize(img, (832, 512))
loaded_dict_enc = torch.load(
    encoder_path, map_location="cuda", weights_only=True
)
filtered_dict_enc = {
    k: v for k, v in loaded_dict_enc.items() if k in encoder.state_dict()
}
encoder.load_state_dict(filtered_dict_enc)
encoder.to("cuda")
encoder.eval()
print("Loading pretrained decoder")
loaded_dict = torch.load(
    depth_decoder_path, map_location="cuda", weights_only=True
)
depth_decoder.load_state_dict(loaded_dict)
depth_decoder.to("cuda")
depth_decoder.eval()
img = torch.tensor(img).to("cuda").permute(2, 0, 1).unsqueeze(0)
 print(input_color.shape)
with torch.no_grad():
    output_dict = depth_decoder(encoder(input_color))
pred_disp, _= disp_to_depth(
    output_dict[("disp", 0)], 0.1, 100.0
)
pred_disp = pred_disp.cpu()[:, 0].numpy()
N = pred_disp.shape[0] // 2
pred_disp = batch_post_process_disparity(pred_disp[:N], pred_disp[N:, :, ::-1])

pred_depth = 1 / pred_disp
pred_depth[pred_depth < MIN_DEPTH] = MIN_DEPTH
pred_depth[pred_depth > MAX_DEPTH] = MAX_DEPTH
pred_depth = pred_depth.transpose(1, 2, 0)
print(pred_depth.min(), pred_depth.max())
cv2.imwrite("img.png", `pred_depth)

**Comment by GenkiK** [comment]
Hi @remy-byte 

Please normalize input image. We normalize it with `torchvision.transforms.v2.ConvertDtype()` in `base_eval.py`. The link to the code is [here](https://github.com/kyotovision-public/fumet/blob/5866cf0f399402a26dcd1e93a179a0acebe0a1b2/datasets/base_eval.py#L48).

**Comment by remy-byte** [comment]
Oh, I wasn't aware of that detail. I'll read the documentation carefully for that function and then come back with a result. Thanks @GenkiK .  :D
